workspaces/ui-v2/src/App.js

```diff
@@ -70,9 +70,8 @@ function AppError(props) {
             or <a href={SupportLinks.GithubIssues}> open a GitHub issue</a>
             {': '}
             <ul>
-              <li>Open Developer Tools</li>
               <li>
-                Type <code>debugOptic()</code> in the console
+                share a debug capture <code>api debug:capture</code>
```

Comment:
This won't be appropriate for the cloud viewer, but we can address that when we bring back the debug dumper